modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/item_finder/AddOrCreate.js

```diff
@@ -30,9 +30,10 @@ function Item(props) {
 		<ClayTable.Row>
 			{props.fields.map((field, i) => {
 				const value = getValueFromItem(props.itemData, field.fieldName);
-				const DataRenderer = resolveRendererByType(
-					field.contentRenderer
-				);
+
+				const DataRenderer =
+					DATA_RENDERERS[field.contentRenderer]?.component ||
+					DATA_RENDERERS.default.component;
```

Comment:
Why is there a `.default` here? Isn't that a named export?

Reply:
I'm am not sure what you mean, `default` here does not have any special meaning, it's not a keyword. It's just an object property called `default`. A renderer of `default` type. And we expect this to be always set, so I think it's safe to omit optional chaining.

Reply:
> A renderer of default type.

Ah I see. I was thinking this was an issue from ESM/Commonjs and you were calling the "default" export of something.